recipes/arrow/all/conanfile.py

```diff
@@ -26,7 +26,7 @@ class ArrowConan(ConanFile):
         "parquet": ["auto", True, False],
         "substrait": [True, False],
         "skyhook": [True, False],
-        "plasma": [True, False],
```

Comment:
See https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-a-recipe for the guideline on how to handle cases like this :)

Reply:
Thanks @RubenRBS !